internal/command/show.go

```diff
@@ -128,6 +130,7 @@ Other options:
   -no-color           Disable terminal escape sequences.
 
   -json               Show the information in a machine-readable form.
+                     Required when using -config.
```

Comment:
I think we can probably skip this just to keep things relatively concise, since the `-config` option already states that it requires `-json` and the command itself will return a clear error if someone tries to use `-config` without `-json`.